### PR TITLE
add network_id based on ansible vars file

### PR DESCRIPTION
**Pull request by valerifilipov**
Regarding https://www.pivotaltracker.com/story/show/164864896

### CODE REVIEW

File: ansible/monitoring.yml [hunk 1]
```diff
@@ -83,6 +89,7 @@
         - "flavor:c{{ ansible_processor_vcpus }}.m{{ (ansible_memtotal_mb/1024)|round|int }}"
         - "public_ipv4:{{ public_ipv4 }}"
         - "role:{{ node_role|default('unknown') }}"
+        - "network_id:{{ node_config.fork_management.network_id|default('unknown') }}"
```
Comment (dincho) [inline ansible/monitoring.yml]: I think this will fail if there is no fork_management defined

Comment (michalbagrowski) [reply ansible/monitoring.yml]: good catch, i will have same issue with backup template file.

**Comment by valerifilipov** [comment]
Fails with Mandatory variable ''db_version'' not defined

File: ansible/monitoring.yml (hunk 1 above)
Comment (valerifilipov) [reply ansible/monitoring.yml]: fixed in 7b0b3dd

**Pull request merged**